io-ballerina/src/io/record/writable_record_channel.bal

```diff
@@ -23,16 +23,15 @@ public class WritableTextRecordChannel {
     private string rs;
 
     # Constructs a DelimitedTextRecordChannel from a given WritableCharacterChannel.
-
     # + characterChannel - The `WritableCharacterChannel`, which will point to the input/output resource
     # + fs - Field separator (this could be a regex)
     # + rs - Record separator (this could be a regex)
     # + fmt - The format, which will be used to represent the CSV (this could be 
     #         "DEFAULT" (the format specified by the CSVChannel), 
     #         "CSV" (Field separator would be "," and record separator would be a new line) or else
     #         "TDF" (Field separator will be a tab and record separator will be a new line). 
```

Comment:
Cant we use enums for this?

Reply:
This should be a separate discussion. I didn't actually cover that in this PR. If so, that will be a breaking change. Let's add an issue and track this separately.

Reply:
Ack